### ISSUE TITLE
"Token is invalid" or "Token does not exist"

### ISSUE THREAD

**Issue by MadMartian** [SKCraft/Launcher#420]
Just tried to login to Minecraft via the launcher today and I am getting _"Token is invalid"_ or _"Token does not exist"_ error pop-up and the game doesn't launch.  It was working yesterday so I assume something has changed on the server side.  My account is a Mojang one, it has not been migrated yet nor is there any indication or way for me to migrate it to Microsoft.  I do not think this is related though because I can login via the Mojang launcher without issues.

I tried logging in to the SKCraft Launcher from my GitHub fork and it _does works_ there, so it is not a server error but a client side state error.  I went back to the troublesome launcher, removed my Mojang account by _forgetting_ it, then I re-added it.  I was able to login this time.

As for troubleshooting information I do not have much, I ran `find ~/.mmm-launcher -cmin -3` and (after ommitting the instance-specific mod files) I note that the following files were modified since logging in successfully (i.e. under 3 minutes ago):

```
./temp/extract
./temp/extract/1618768707848
./temp/extract/1618768707848/liblwjgl64.so
./temp/extract/1618768707848/liblwjgl.so
./temp/extract/1618768707848/libjinput-linux.so
./temp/extract/1618768707848/libjinput-linux64.so
./temp/extract/1618768707848/libopenal.so
./temp/extract/1618768707848/libopenal64.so
./accounts.dat
./instances/MMM/instance.json
```

Our launcher is a few commits branched off of SKCraft at revision `daa955fac6618be5d880333c3ee162aea46e21b3` - _"Remove packed check from launcher self-updater"_

**Comment by hedgehog1029** [comment]
Yeah, that's expected. Basically it means that the session has been completely invalidated and you need to log in again. The way Yggdrasil handles multiple sessions is arcane and baffling; I don't anticipate this happening with Microsoft auth, but we'll see.

What I *should* do is pop up the login window to indicate "hey, you just need to log in again, no big deal" especially since the current error isn't very informative. But yeah, right now if you get that error you just need to remove and re-add the account.

**Comment by hedgehog1029** [comment]
Fixed in b7be91bc9cc9421075806d3cecf28b00e4ec92ca :)
